packages/orbit-components/src/InputField/InputField.stories.tsx

```diff
@@ -10,236 +11,236 @@ import ServiceLogo from "../ServiceLogo";
+      exclude: ["onChange", "prefix", "suffix"],
+    },
   },
-};
 
-EmailInput.args = {
-  label: "Email",
-  value: "name@example.co",
-  placeholder: "Email",
-};
-
-export const WithTextPrefix = ({ label, value, placeholder, prefix }) => {
-  return (
-    <InputField
-      label={label}
-      value={value}
-      placeholder={placeholder}
-      prefix={prefix}
-      onChange={action("change")}
-    />
-  );
+  args: {
+    type: TYPE_OPTIONS.EMAIL,
+    label: "Email",
+    value: "name@example.co",
+    placeholder: "Email",
+  },
 };
 
-WithTextPrefix.story = {
-  name: "With text prefix",
-
+export const WithTextPrefix: Story = {
   parameters: {
-    info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: ["onChange", "suffix"],
+    },
   },
-};
 
-WithTextPrefix.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-  prefix: "$",
-};
+  args: {
+    label: "Label",
+    value: "",
+    placeholder: "Placeholder",
+    prefix: "$",
+  },
 
-export const WithTextSuffix = ({ label, value, placeholder, suffix }) => {
-  return (
-    <InputField
-      label={label}
-      value={value}
-      placeholder={placeholder}
-      suffix={<div style={{ paddingRight: "12px" }}>{suffix}</div>}
-      onChange={action("change")}
-    />
-  );
+  argTypes: {
+    prefix: {
+      control: {
+        type: "text",
+      },
+    },
+  },
 };
 
-WithTextSuffix.story = {
-  name: "With text suffix",
+export const WithTextSuffix: Story = {
+  render: ({ suffix, ...args }) => (
+    <InputField suffix={<div style={{ paddingRight: "12px" }}>{suffix}</div>} {...args} />
+  ),
 
   parameters: {
-    info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: ["onChange", "prefix"],
+    },
   },
-};
 
-WithTextSuffix.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-  suffix: "Some long text",
-};
+  args: {
+    suffix: "Some long text",
+  },
 
-export const CompactInput = ({ value, label, placeholder, required, error }) => {
-  return (
-    <InputField
-      label={label}
-      inlineLabel
-      error={error}
-      value={value}
-      placeholder={placeholder}
-      required={required}
-      onChange={action("change")}
-    />
-  );
+  argTypes: {
+    suffix: {
+      control: {
+        type: "text",
+      },
+    },
+  },
 };
 
-CompactInput.story = {
-  name: "Compact input",
-
+export const CompactInput: Story = {
   parameters: {
     info: "Compact input with FormLabel as prefix",
+    controls: {
+      exclude: ["onChange", "prefix", "suffix"],
+    },
   },
-};
 
-CompactInput.args = {
-  value: "",
-  label: "Label",
-  placeholder: "Placeholder",
-  required: false,
-  error: "",
+  args: {
+    inlineLabel: true,
+  },
 };
 
-export const CompactInputWithTags = ({ value, label, placeholder, required, error }) => {
-  return (
+export const CompactInputWithTags: Story = {
+  render: args => (
```

Comment:
When having `required=true`, `inlineLabel=true` and help input filled in, I don't see the help icon in the input. Is it expected for this type of input field or is it a bug (it is not happening for other stories)?

<img width="362" alt="image" src="https://github.com/user-attachments/assets/1f9aa1c3-fbd7-4b4f-912e-0a3457bfe3e1">

number input story:
<img width="317" alt="image" src="https://github.com/user-attachments/assets/54a7e4f1-a73d-49a6-b36a-77847fd6aa52">

Reply:
Let's wait for the information in [Slack thread](https://skypicker.slack.com/archives/C05FDP4UM3N/p1723638350535459) and then I think PR is ok. 😃